config/zones.json

```diff
@@ -1524,12 +1531,39 @@
     ]
   },
   "US-NY": {
+    "capacity": {
```

Comment:
If I'm not mistaken we are missing a reference to US-NY capacities in the README.

Reply:
Still needs to be addressed

Reply:
long story short, still working on

Reply:
it's on its way in commit 271d460 (below)

Sorting how I seem to have painted myself into a corner creating a branch "add-PJM-Parser" 5 days ago. ~hans